content/influxdb/cloud-dedicated/admin/cluster-data.md

```diff
@@ -0,0 +1,398 @@
+---
+title: Query cluster system data
+description: >
+  Query system tables in your InfluxDB Cloud Dedicated cluster to see data related
+  to queries, tables, partitions, and compaction in your cluster.
+menu:
+  influxdb_cloud_dedicated:
+    parent: Administer InfluxDB Cloud
+    name: Query cluster system data
+weight: 105
+---
+
+{{< product-name >}} stores data related to queries, tables, partitions, and
+compaction in system tables in your cluster.
+Query data in your cluster's system tables for information about your cluster.
+
+- [Query system tables](#query-system-tables)
+- [System tables](#system-tables)
+- [System query examples](#system-query-examples)
+
+{{% warn %}}
+#### May impact overall cluster performance
+
+Querying InfluxDB v3 system tables may impact the overall write and query
+performance of your {{< product-name omit=" Clustered" >}} cluster.
+
+<!--------------- UPDATE THE DATE BELOW AS EXAMPLES ARE UPDATED --------------->
+
+#### System tables are subject to change
+
+System tables are not part of InfluxDB's stable API and may change with new releases.
+The provided schema information and query examples are valid as of **April 11, 2024**.
+If you detect a schema change or a non-functioning query example, please
+[let us know](https://github.com/influxdata/docs-v2/issues/new/choose).
+
+<!--------------- UPDATE THE DATE ABOVE AS EXAMPLES ARE UPDATED --------------->
+{{% /warn %}}
+
+## Query system tables
+
+{{% warn %}}
+_Querying system tables [may impact overall cluster performance](#may-impact-overall-cluster-performance)._
+{{% /warn %}}
+
+{{% note %}}
+Querying system tables requires **`influxctl` v2.8.0 or newer**.
```

Comment:
But, it doesn't require `influxctl`, right? You can still use the `iox-debug` header?

Reply:
True. What about:

```suggestion
Querying system tables with `influxctl` requires **`influxctl` v2.8.0 or newer**.
```